test/integration_tests/conftest.py

```diff
@@ -73,3 +73,13 @@ def temp_hub_dir(tmpdir, pytestconfig):
         torch.hub.set_dir(tmpdir)
         yield
         torch.hub.set_dir(org_dir)
+
+
+@pytest.fixture()
+def emissions():
```

Comment:
I would go one step further and  use the actual emission.

1. Pick a license free speech
2. Feed it to wav2vec2 model
3. Save the resulting emission
4. Upload it to download.pytorch.org/test-assets/
5. Fetch it at the test time.